chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEvent.kt

```diff
@@ -1,12 +1,13 @@
 package com.pusher.chatkit.cursors
 
+import com.pusher.chatkit.cursors.api.CursorApiType
 import elements.Error
 
-sealed class CursorSubscriptionEvent {
-    data class OnCursorSet(val cursor: Cursor) : CursorSubscriptionEvent()
-    data class InitialState(val cursors: List<Cursor>) : CursorSubscriptionEvent()
+internal sealed class CursorSubscriptionEvent {
+    data class OnCursorSet(val cursor: CursorApiType) : CursorSubscriptionEvent()
+    data class InitialState(val cursors: List<CursorApiType>) : CursorSubscriptionEvent()
     data class OnError(val error: Error) : CursorSubscriptionEvent()
     object NoEvent : CursorSubscriptionEvent()
```

Comment:
NoEvent can be removed

Reply:
It's currently used by the [CursorEventParser for cursor_set ](https://github.com/pusher/chatkit-android/blob/master/chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEventParser.kt#L35) - what should we do there instead?

Reply:
It can be just removed from the both places, likewise OnError (which I referred to in the other [comment](https://github.com/pusher/chatkit-android/pull/179/files#r380261922)).

Reply:
Ah, I had asked in the other comment which line you were referring to - the error is still in use by the parser [here](chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEventParser.kt) on L38

Reply:
yep, the error creation can be removed from the parser, more [elaboration](https://github.com/pusher/chatkit-android/pull/179#discussion_r380261922)

Reply:
👍 have removed.